### PR TITLE
docs: add Web Console to Query Data page

### LINKED ISSUE #187: Add console at the top of the Query data section

### PR DESCRIPTION
**Pull request by bsmth**
__Description:__
* Adds new Web Console section to Query data with example self-contained SQL that will create data

closes #187

### CODE REVIEW

File: docs/develop/query-data.md [hunk 1]
```diff
@@ -1,26 +1,79 @@
...
-This page shows how to query data from QuestDB using different programming
-languages or tools.
+This page describes how to query data from QuestDB using the embedded Web
+Console, make requests towards the REST API endpoints that QuestDB exposes, and
+Postgres wire protocol via multiple programming languages.
 
 ## Prerequisites
 
-Make sure you have QuestDB running and accessible, you can do so from
+QuestDB must running and accessible, you can do so from
```
Comment (mpsq) [inline docs/develop/query-data.md:14]:
```suggestion
QuestDB must be running and accessible, you can do so from
```

File: docs/develop/query-data.md [hunk 2]
```diff
@@ -1,26 +1,79 @@
...
+
+Aside from the Code Editor for writing and executing SQL queries, the Web
+Console has the following additional components:
+
+- A Schema Explorer which displays tables and their schemas
+- A Visualization panel for viewing query results as tables or graphs
+- An Import tab for uploading datasets as CSV files
+
+For more details on these components and general use of the console, see the
+[Web Console reference](/docs/reference/client/web-console/) page
```
Comment (mpsq) [inline docs/develop/query-data.md:67]:
```suggestion
[Web Console reference](/docs/reference/client/web-console/) page.
```

**Review by mpsq — CHANGES_REQUESTED** [review]
minor tweaks

File: docs/develop/query-data.md (hunk 1 above)
Comment (bsmth) [reply docs/develop/query-data.md:14]: good catch

**Pull request merged**